src/annotator/test/bucket-bar-test.js

```diff
@@ -1,34 +1,27 @@
-import BucketBar from '../bucket-bar';
-import { $imports } from '../bucket-bar';
+import BucketBar, { $imports } from '../bucket-bar';
```

Comment:
At some point we should convert BucketBar to a named export.

Reply:
> But yes, we can be more strict, and everything that is not in a components folder could be converted to a named export. That would include Sidebar, Guest and others.

I think we should do this. Applying the strict rule that only Preact components, from a file of the same name, use default exports, is in line with our [JS style guide](https://github.com/hypothesis/frontend-toolkit/blob/master/docs/js-guide.md#module-naming) and is an easier convention for future devs to follow.